x-pack/qa/multi-cluster-search-security/legacy-with-basic-license/src/test/resources/rest-api-spec/test/fulfilling_cluster/10_basic.yml

```diff
@@ -127,6 +127,8 @@ setup:
               properties:
                 '@timestamp':
                   type: date
+            settings:
+              'index.number_of_shards': 1
```

Comment:
why change number of shards?

Reply:
I need to assert the number of shards that search requests with data streams executed against. The default the number of shards is 1; but we might random it in YAML tests.